README.md

```diff
@@ -56,7 +56,7 @@ For More Info on what a FOSS Contributor Fund is, and how to start your own, [Wa
 > My GitHub ID is:,Email Address  
 > danisyellis,dgellis@indeed.com  
 > octocat,octocat@github.com  
-> thisShouldError,notaname@example.com
+> thisShouldError,notARealGithubId@example.com
```

Comment:
I changed this so the intention is more clear.
I'm also not entirely sure I need to have an example that errors in the example CSV. But I have found it useful for my own test code. 
Also, if I don't use that, I'd have to figure out a third github Id I feel comfortable using as an example (maybe Duane?) because I don't think an example with only 2 entries is as clear as an example with 3.